File: Vachagan/CS50x 2019/Week 1/cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main()
{
    int coins;

    do
    {
        coins = round(get_float("Change owed: ") * 100);
    }
    while (coins <= 0);
    
    printf("%i\n", coins / 25 + (coins % 25) / 10 + ((coins % 25) % 10) / 5 + ((coins % 25) % 10) % 5);
}

File: Vachagan/CS50x 2019/Week 6/cash/cash.py
from cs50 import get_float

coins = 0
while (coins <= 0):
    coins = get_float("Change owed: ") * 100

print(coins // 25 + (coins % 25) // 10 + ((coins % 25) % 10) // 5 + ((coins % 25) % 10) % 5)
